layer-atlas/src/main/res/values/atlas-colors.xml

```diff
@@ -26,6 +26,8 @@
     <color name="atlas_delete_background">#FFFF5252</color>
     <color name="atlas_item_background">@android:color/white</color>
 
+    <color name="atlas_app_bar_background">#20000000</color>
```

Comment:
Shouldn't this be in the app primary colour?

Reply:
For that specific case I didn't see it necessary since the ImageView will be full screen size and I just wanted to have the ActionBar to be translucent.
If you think we should apply the primarycolor and then change it to be translucent I'm ok with the idea.

Reply:
👍